.travis.yml

```diff
@@ -63,16 +62,15 @@ before_script:
   # Remember the Drupal installation path.
   - DRUPAL_DIR=$(pwd)/drupal
 
-  # Download Drupal console so we can run the test for it. Skip this for the
-  # coding standards test.
-  - test ${TEST_SUITE} == "PHP_CodeSniffer" || composer require --dev drupal/console:~1.0 --working-dir=$DRUPAL_DIR
-
   # Install Composer dependencies for core. Skip this for the coding standards test.
   - test ${TEST_SUITE} == "PHP_CodeSniffer" || composer install --working-dir=$DRUPAL_DIR
 
   # PHPUnit 6 is required when running tests on PHP 7.x.
   - test ${TEST_SUITE} == "PHP_CodeSniffer" || test ${TRAVIS_PHP_VERSION:0:3} == "5.6" || composer require --dev phpunit/phpunit:~6 --update-with-dependencies --working-dir=$DRUPAL_DIR
 
+  # PHPUnit 7 is required when running tests Drupal 9.
+  - test ${TEST_SUITE:0:1} == "9"  || composer require --dev phpunit/phpunit:~7 --update-with-dependencies --working-dir=$DRUPAL_DIR
```

Comment:
Change the `||` to a `&&` and it should work.

Reply:
```suggestion
  - test ${TEST_SUITE:0:1} == "9"  && composer require --dev phpunit/phpunit:~7 --update-with-dependencies --working-dir=$DRUPAL_DIR
```